docs/serving/gettingstarted/README.md

```diff
@@ -0,0 +1,5 @@
+This guide provides information about getting started with deploying and managing Knative applications, and optional configuration settings that you can modify for your Knative Serving deployment.
```

Comment:
This seems like a pretty short page. Should this be at the start of `docs/serving/README.md`, or is this part of a reorganization of content?